src/core/trulens/core/schema/app.py

```diff
@@ -352,6 +352,8 @@ def _submit_feedback_functions(
         tp = TP()
 
         for ffunc in feedback_functions:
+            ffunc.run(app=app, record=record)
```

Comment:
Why is this needed?

Reply:
Ah some working changes got checked in. Will revert this